backend/controllers/encryptionController.js

```diff
@@ -0,0 +1,97 @@
+const axios = require('axios').default;
+const cloudinary = require('cloudinary');
+const bcrypt = require('bcryptjs');
+const sjcl = require('sjcl');
+const internalServerError = require('../utils/internalServerError');
+const connection = require('../config/database');
+const { INSERT_ENCRYPTION } = require('../services/encryptionServices');
+
+const { STABLE_DIFFUSION_VERSION: version, REPLICATE_API_TOKEN: token } = process.env;
+
+exports.encryptMessage = async (req, res) => {
+  const {
+    sharkName, secretMessage, secretKey, selectedStyle,
+  } = req.body;
+
+  if ((!sharkName || !secretMessage || !secretKey, !selectedStyle)) {
```

Comment:
Hey, can you please check the condition here.

Reply:
Will do. Thanks for your review!